kong/workspaces/init.lua

```diff
@@ -48,5 +48,8 @@ function workspaces.get_workspace_id(ctx)
   return (ctx or ngx.ctx).workspace or kong.default_workspace
 end
 
+function workspaces.get_workspace_name()
+    return (workspaces:get_workspace() or { name = "default" }).name
```

Comment:
Please do not create a new table on the fly.

Reply:
> Please do not create a new table on the fly.

@hbagdi I looked at this and was not sure if I thought it was clever or ugly, but I was not aware that we have a rule against it.  What is the reason for not doing it this way?

Reply:
If tables are static in nature then creating them on the fly causes unnecessary GC pressure. I'm out of depth with LuaJIT and I'm not sure if the compiler optimizes this specific instance away or not, I prefer being conservative.